src/utils/formatMidiMessage.ts

```
@@ -47,7 +48,65 @@ interface DeviceEnquirySysexMidiMessage extends BaseSysexMidiMessage {
+   clockRateLo: number;
+
+   mwLevel: number;
+   atLevel: number;
+   keyTransposeDisable: number;
+   displayMode: number;
+   memoryProtect: number;
+   internalTune: number;
+   externalPedalPolarity: number;
+}
+
+interface Kiwi106PatchDumpSysexMidiMessage extends BaseSysexMidiMessage {
```

ESLint

[error] 'Kiwi106PatchDumpSysexMidiMessage' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)